### PR TITLE
make the pgp check more strict

### PR DESCRIPTION
**Pull request by amyulating**

### CODE REVIEW

**Review by SammCheese — COMMENTED** [review]
Impov

File: src/moderate.ts [hunk 1]
```diff
@@ -68,13 +68,11 @@ export async function moderateMessage(msg: Message) {
...
         if (a.filename.endsWith(".txt")) {
-            const content = await fetch(a.url, {
-                headers: {
-                    Range: "bytes=0-13"
-                }
-            }).then(r => r.text());
+            const content = await fetch(a.url).then(r => r.text());
 
-            if (content === "-----BEGIN PGP") {
+            const pgpRegex = /pgp/i;
```
Comment (SammCheese) [inline src/moderate.ts]: more specific
```suggestion
            const pgpRegex = /BEGIN PGP/i; 
```

**Comment by amyulating** [comment]
im currently looking for a more strict way

**Comment by SammCheese** [comment]
stop

**Review by SammCheese — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@exhq loves the [Scunthorpe problem](https://en.wikipedia.org/wiki/Scunthorpe_problem)

File: src/moderate.ts (hunk 1 above)
Comment (amyulating) [reply src/moderate.ts]: this doesnt really solve the problem that *you can literally fucking delete the header*

Comment (SammCheese) [reply src/moderate.ts]: ok and?

**Comment by Vendicated** [comment]
https://github.com/Vendicated/Vaius/commit/e28254e4785e6b7737020dea0447ef46f1c85ace

**Pull request closed without merging**